wazo_dird/plugins/default_json/api.yml

```diff
@@ -29,6 +29,37 @@ paths:
       - $ref: '#/parameters/tenantuuid'
       - $ref: '#/parameters/Profile'
       - $ref: '#/parameters/Term'
+  /directories/lookup/{profile}/{xivo_user_uuid}:
+    get:
+      summary: Search for contacts for a particular user
+      description: '**Required ACL:** `dird.directories.lookup.{profile}.{xivo_user_uuid}.read`
+
+
+        The `lookup` query will return a list of result matching the searched term.
+        The result will be retrieved from all configured directories for the given
+        profile and user.
+
+
+
+        This route is provided by the `default_json_view` plugin using the `lookup`
+        plugin and all configured sources for the given profile.'
+      operationId: lookup
```

Comment:
I think the `operationId` needs to be different for all resources in the spec

Reply:
From the swagger editor
```
Semantic error at paths./directories/lookup/{profile}/{xivo_user_uuid}.get.operationId
Operations must have unique operationIds.
```
Unfortunately there are so many errors that its hard to see when we introduce new errors